About schema.org

Morban15 opened the pull request.

Morban15 committed:
About schema.org

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+
+`itemprop` especifica las propiedades del elemento. 
+
+===============================================================================
```
Markdown has a way to generate horizontal rules, `----`. 

----

But I think that instead of using horizontal rules, we can use headers (h2 for the main sections) to break apart the work and it will help readability of the document. 

What I read the document, I sense 3 sections 

## What is schema.org?

## Tips for implementing/using schema.org

## Testing schema.org

What do you think?

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+
+Vamos a utilizar un ejemplo:
+
+![schema](images/schema.png)
```
I was expecting to see a screenshot about a schema.org type of work. I think it would be helpful for people to first see an example of a work that can be marked up with schema, article or news are common enough. 

Then you can draw a correlation between the property types and actual examples from the docs because it is also important to know how to read the documentation and determine which `itemscope` you want to use.

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+Ir a [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool)
+para verificar que el `schema.org` está correctamente implementado. 
+
+Para más información visita [Schema.org](https://schema.org)
```
I think it would be good to include the article you all read about structured data as a resource, too.

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+
+========================================================================
+
+Ir a [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool)
```
This section on testing could be built out to help readers understand what a passing test looks like vs what a failing test looks like, and how to fix the results.

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+![examplePost](images/examplePost.png)
+
+![examplePost2](images/examplePost2.png)
+
```
It would also be good to talk a bit about how sometimes you have to add extra markup, such as inline elements, such as `span` to provide a wrapper for `itemprop`. The before and after example for the nav might be worth highlighting to visually show this

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+el contenido que sea visible para las personas que visitan la página 
+web y no el contenido en `div` ocultos u otros elementos ocultos.
+
+2. A veces el valor de una propiedad de elemento puede ser otro elemento con su propio conjunto de propiedades.
```
What do you mean here? Can you show this?

misscs commented on schema.md:
```diff
@@ -0,0 +1,43 @@
+Schema.org describe una variedad de tipos de elementos, cada uno tiene su conjunto de propiedades que se pueden utilizar para describir el elemento. 
+Ir a [Lista de todos los tipos de elementos](https://schema.org/docs/full.html) para ver los detalles.
+
+**Algunas notas a tener en cuenta a la hora de utilizar schema.org:**
```
This is probably best marked up as `h3` (###)

misscs reviewed (changes requested):
Let me know if my suggestions make sense. This is probably one of the more difficult guides because it's dense and a complex subject matter.

Morban15 committed:
Guide divided into three sections

Morban15 committed:
Work in progress

Morban15 committed:
Replacing and renaming images

Morban15 committed:
Working in the example itemscope part in the first section

Morban15 committed:
Working in the itemtype for the example

Morban15 committed:
Working in the itemprop part in the example

Morban15 committed:
Deleting unnecessary imgs and codes from itemprop part

Morban15 committed:
Giving an example for the tip number 2

Morban15 committed:
Remaking the third tip example

Morban15 committed:
Working in the last section from the guide

Morban15 committed:
Cleaning

misscs committed:
Add link to scheme.org

misscs committed:
Adjust itemscope content to avoid duplication

There was a bit of redundancy between the sentences so I merged them together.

misscs committed:
Restructure itemtype documentation

misscs committed:
Restrucure itemprop documentation

misscs committed:
Consolidate headers

Both headers say the same thing differently. I went with the second one because I think the more conversational tone fits the article

misscs committed:
Remove 'Pero' at the start of the sentence

Spacing adjustments (I bet our editors use different settings ak)

misscs committed:
Cleanup - markdown linter alerts

misscs committed:
Convert to headers and lists

Headers and lists are easier to parse. If I'm using bold to indicate a section title, it is usually a sign it can likely use a header element instead.

misscs committed:
Use the imperative tense

misscs reviewed (approved):
I think we have a good base for now. Thanks for patiently waiting on my changes.
